sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormContent.java

```diff
@@ -26,25 +26,17 @@
      */
     private final BoundingBox boundingBox;
 
-    /*
-     * Form text content type.
-     */
-    private final TextContentType textContentType;
-
     /**
      * Creates raw OCR item.
      *
-     * @param text The text content of ExtractedField.
-     * @param boundingBox The BoundingBox of ExtractedField.
+     * @param text The text content of the field.
+     * @param boundingBox The BoundingBox specifying relative coordinates of the field.
```

Comment:
Might be nice to be more specific for bounding box here. Here's what python says:
"A list of 4 points representing the quadrilateral bounding box that outlines the text. The points are listed in clockwise order: top-left, top-right, bottom-right, bottom-left. Units are in pixels for images and inches for PDF."

Reply:
Are we currently calling this out as 4 points, I was under the impression this could be more than 4 points?

Reply:
From my understanding it is only 4 points. Here's what the service says:

Quadrangle bounding box of a line or word, depending on the parent object, specified as a list of 8 numbers. The coordinates are specified relative to the top-left of the original image. The eight numbers represent the four points, clockwise from the top-left corner relative to the text orientation. For image, the (x, y) coordinates are measured in pixels. For PDF, the (x, y) coordinates are measured in inches.

Reply:
I think it is a future prospect for them to grow to have more than 4 points from the service side. 
But I think we should be good for now.

cc: @annelo-msft